### PR TITLE
Add mapping bussiness models workshop

### PR DESCRIPTION
**Pull request by clausmullie**
Works towards #685 

-----
[View rendered activities/workshops/index.md](https://github.com/publiccodenet/about/blob/add-map-bussiness-workshop/activities/workshops/index.md)
[View rendered activities/workshops/mapping-business-models/agenda-template.md](https://github.com/publiccodenet/about/blob/add-map-bussiness-workshop/activities/workshops/mapping-business-models/agenda-template.md)
[View rendered activities/workshops/mapping-business-models/canvas-template.md](https://github.com/publiccodenet/about/blob/add-map-bussiness-workshop/activities/workshops/mapping-business-models/canvas-template.md)
[View rendered activities/workshops/mapping-business-models/index.md](https://github.com/publiccodenet/about/blob/add-map-bussiness-workshop/activities/workshops/mapping-business-models/index.md)
[View rendered activities/workshops/mapping-business-models/workshop-guide.md](https://github.com/publiccodenet/about/blob/add-map-bussiness-workshop/activities/workshops/mapping-business-models/workshop-guide.md)

### CODE REVIEW

**Comment by clausmullie** [comment]
@felixfaassen @Ainali before I start loading in content, what do you think of this structure and placement?

File: activities/workshops/index.md [hunk 1]
```diff
@@ -8,5 +8,6 @@ Here is a list of workshops developed or used by the Foundation for Public Code:
 
 * [Governance exercise](../supporting-codebase-governance/exercise/index.md)
 * [Governance game](../supporting-codebase-governance/game/index.md)
+* [Mapping business models enabled by codebase](mapping-business-models.md)
```
Comment (felixfaassen) [inline activities/workshops/index.md:11]: * fix the link in the index.md of the workshop folder pointing to the mapping-business-models folder (link is broken)

Comment (clausmullie) [reply activities/workshops/index.md]: Done!

**Comment by bvhme** [comment]
I think it is worth thinking as part of what 'activity' this should live under. What is this thing we do to help?

Is this a part of 'supporting codebase governance' or something else?

The codebase stewardship activities page mentions 'support with vendor relations, including maintenance, implementation and integration partners as well as hosting providers.' Which I feel is in the right direction but maybe also not really what we're looking for. Publiccode.net speaks of 'We also help vendors such as cooperatives, NGOs and for profit companies to sell, implement and support the product.' which I also think is in the right direction but not the right thing.

Is there some snappy thing we can come up with. I was thinking of market-making and then realized how bad that sounds ;)

**Comment by clausmullie** [comment]
I'm fairly happy with 'a workshop' living under the activity 'Workshop'.

This workshop could be used in codebase stewardship, as community building or support stewardship, but it could just as well be used in membership, making arguments how a codebase/open source can enable (new) markets and isn't "anti-market". This is a tool, up to the user on how to use it.

See for reference https://github.com/publiccodenet/about/pull/649#issue-432033363

I think it should live here, and we can link back to it from where it has been identified to create value.

**Comment by clausmullie** [comment]
@bvhme - should this be merged as is, or do you have a suggested fix for the whitespaces? If we remove them, the agenda looks garbled.

**Review by Ainali — APPROVED** [review]

**Pull request merged**